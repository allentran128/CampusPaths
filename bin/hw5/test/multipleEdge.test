# Create a graph
CreateGraph graph1

# Add nodes
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 a1

# Add edges
AddEdge graph1 n1 n2 e1
AddEdge graph1 n1 a1 e2
AddEdge graph1 n2 n1 e3

# Print Graph
ListNodes graph1

# List Children of n1
ListChildren graph1 n1
