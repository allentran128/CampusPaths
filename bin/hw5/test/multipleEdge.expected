# Create a graph
created graph graph1

# Add nodes
added node n1 to graph1
added node n2 to graph1
added node a1 to graph1

# Add edges
added edge e1 from n1 to n2 in graph1
added edge e2 from n1 to a1 in graph1
added edge e3 from n2 to n1 in graph1

# Print Graph
graph1 contains: a1 n1 n2

# List Children of n1
the children of n1 in graph1 are: a1(e2) n2(e1)